.bazelversion

```diff
@@ -0,0 +1 @@
+7.2.0rc1
```

Comment:
Is this necessary? I'd rather do 7.1.2 unless there's feature we absolutely require.

Reply:
It's not, but we'll have to add MODULE.bazel.lock to .gitignore since it wasn't stable

Reply:
OK, we can start there at least, if it's all the same for you? :)